init.js

```diff
@@ -203,7 +203,12 @@ var spawnPoolWorkers = function(){
         } else if (!connection) {
             redisConfig = pcfg.redis;
             connection = redis.createClient(redisConfig.port, redisConfig.host);
-            connection.auth(redisConfig.password);
+            if (redisConfig.password != "") {
```

Comment:
We must do either check (redisConfig.password != "") or remove "password" from config.json (lines 57 and 29) because default password is "".

with this check we don't allow users to use empty password and will keep conf.json with fill example config.